docs/microsoft-365-developer-program-faq.yml

```diff
@@ -136,7 +138,7 @@ sections:
       - question: |
           How can I get a new subscription?
         answer: |
-          You can only have one Microsoft 365 developer subscription associated with your Developer Program account. If you want to get a new instant or configurable sandbox, you can [delete your account](#how-many-microsoft-365-developer-program-accounts-can-i-create-) and set up a new one. Note that you can only get a new subscription after you have had your subscription for more than 60 days.
+          If you're a qualified program member, you're entitled to one Microsoft 365 developer subscription associated with your developer program account. If you want to get a new instant or configurable sandbox, you should [delete your account](#how-many-microsoft-365-developer-program-accounts-can-i-create-) and set up a new one. Note that you can only get a new subscription after you've had your current subscription for more than 60 days.
```

Comment:
we need to remove "or configurable" in this case. 
If you want to get a new instant sandbox,